app/controllers/descriptives_controller.rb

```diff
@@ -2,15 +2,13 @@
 
 # Download the CSV descriptive metadata
 class DescriptivesController < ApplicationController
+  before_action :maybe_find_user_version, only: :show
```

Comment:
Decided on this approach due to https://github.com/CanCanCommunity/cancancan/blob/develop/docs/changing_defaults.md#override-loading, but if we already have a pattern we like better, I'd be happy to follow it.